File: Decorator/C#/CoffeeShopNoDecorator/DarkRoastWithMilkAndWhippedCream.cs
namespace CoffeeShopNoDecorator
{
    public class DarkRoastWithMilkAndWhippedCream : Beverage
    {
        public DarkRoastWithMilkAndWhippedCream()
        {
            Description = "Dark roast with milk and whipped cream";
        }

        public override decimal GetPrice()
        {
            return 1.1m + 0.2m + 0.4m;
        }
    }
}

File: Decorator/Java/CoffeeShopNoDecorator/src/CoffeeShopNoDecorator/DarkRoastWithMilkAndWhippedCream.java
package CoffeeShopNoDecorator;

public class DarkRoastWithMilkAndWhippedCream extends Beverage
{
    public DarkRoastWithMilkAndWhippedCream()
    {
        description = "Dark roast with milk and whipped cream";
    }

    @Override
    public double getPrice()
    {
        return 1.1d + 0.2d + 0.4d;
    }
}